pipeline/backend/kubernetes/service.go

```diff
@@ -15,6 +15,8 @@
 package kubernetes
 
 import (
+	"fmt"
+
```

Comment:
Unnecessary newline

Reply:
It should be gone

Reply:
I guess `gofumpt` wants this

```bash
$ cat pipeline/backend/kubernetes/service.go 
...
import (
        "fmt"
        v1 "k8s.io/api/core/v1"
        metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        "k8s.io/apimachinery/pkg/util/intstr"
)
...

$ make lint
Running golangci-lint
golangci-lint run --timeout 15m
pipeline/backend/kubernetes/service.go:18: File is not `gofumpt`-ed with `-extra` (gofumpt)
        "fmt"
make: *** [Makefile:139: lint] Error 1

$ make format 
$ git diff
 import (
        "fmt"
+
        v1 "k8s.io/api/core/v1"
        metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        "k8s.io/apimachinery/pkg/util/intstr"
```

Reply:
https://github.com/woodpecker-ci/woodpecker/pull/2933#discussion_r1429210373